spec/controllers/application_templates_controller_spec.rb

```diff
@@ -112,4 +112,60 @@
       end
     end
   end
+  describe 'POST #toggle_eeo_enabled' do
+    before :each do
+      department = create :department, name: 'Bus'
+      position = create :position, department: department, name: 'Operator'
+      @template = create :application_template, position: position
+    end # end before each
```

Comment:
In general, it shouldn't be needed to comment on end tags in Ruby - better to just keep blocks nice and short.

The only exceptions in this context might be the top-level `describe` block - nothing else here should span more than a few lines at a time.